micro-rdk/src/common/i2c.rs

```diff
@@ -25,7 +25,7 @@ pub enum I2CErrors {
 // embedded_hal I2C trait when supporting boards beyond ESP32.
 pub trait I2CHandle {
     fn name(&self) -> String;
-
+    fn bus_no(&self) -> u8;
```

Comment:
I think I'd prefer either just `bus` or `bus_number`. I also think this should not return a number, but instead an `&str` or a `String`. The reason is that, while most processors will use a numbering scheme for buses, that scheme is not universally enforced. A `String` can most flexibly accommodate different conventions and I think it's ok for board-specific drivers to rely on numbering schemes defined by that board's implementation

Reply:
that works. @stevebriskin and I were talking about maybe removing the `name` portion from an i2c config as it's just an alias and is now an unused remnant, so this would replace that.